src/Sulu/Bundle/ContactBundle/Admin/ContactAdmin.php

```diff
@@ -105,25 +105,40 @@ public function getRoutes(): array
                 ->addOption('title', 'sulu_contact.persons')
                 ->addOption('adapters', ['table'])
                 ->addOption('resourceKey', 'contacts')
-                ->addOption('addRoute', 'sulu_contact.add_form.detail')
-                ->addOption('editRoute', 'sulu_contact.edit_form.detail'),
-            (new Route('sulu_contact.add_form', '/contacts/add', 'sulu_admin.resource_tabs'))
+                ->addOption('addRoute', 'sulu_contact.contact_add_form.detail')
+                ->addOption('editRoute', 'sulu_contact.contact_edit_form.detail'),
+            (new Route('sulu_contact.contact_add_form', '/contacts/add', 'sulu_admin.resource_tabs'))
                 ->addOption('resourceKey', 'contacts'),
-            (new Route('sulu_contact.add_form.detail', '/details', 'sulu_admin.form'))
+            (new Route('sulu_contact.contact_add_form.detail', '/details', 'sulu_admin.form'))
                 ->addOption('tabTitle', 'sulu_contact.details')
                 ->addOption('backRoute', 'sulu_contact.contacts_datagrid')
-                ->addOption('editRoute', 'sulu_contact.edit_form.detail')
-                ->setParent('sulu_contact.add_form'),
-            (new Route('sulu_contact.edit_form', '/contacts/:id', 'sulu_admin.resource_tabs'))
+                ->addOption('editRoute', 'sulu_contact.contact_edit_form.detail')
+                ->setParent('sulu_contact.contact_add_form'),
+            (new Route('sulu_contact.contact_edit_form', '/contacts/:id', 'sulu_admin.resource_tabs'))
                 ->addOption('resourceKey', 'contacts'),
             (new Route('sulu_contact.edit_form.detail', '/details', 'sulu_admin.form'))
```

Comment:
You have forgotten to also update this route name, that's why the edit button in the contacts datagrid is not working correctly.

Reply:
I also realized that there is no nice error shown in case something like that happens, so I quickly implemented that in #3935